app/screens/ProductList/ProductList.js

```diff
@@ -113,21 +115,33 @@ class ProductList extends Component {
 
         return (
             <View style={styles.container}>
-                <Animated.View style={[styles.titleWrap, transformStyle]}>
-                    <Text style={styles.title}>Products</Text>
-                </Animated.View>
-                <AnimatedFlatList
-                    contentContainerStyle={styles.flatListMarginTop}
-                    onScroll={this.onScrollHandlerAnimation}
-                    data={this.state.items}
-                    renderItem={this.renderItems}
-                    keyExtractor={this.keyExtractor}
-                    style={styles.flatList}
-                    refreshing={this.state.isFetching}
-                    onRefresh={this.onRefresh}
-                    onEndReached={this.onScrollHandler}
-                    onEndReachedThreshold={0.5}
-                />
+                { this.state.isGreetingShown ?
+                    <View style={styles.greetingMessage}>
+                        <Text style={styles.title}>{this.getGreetingMessage()}</Text>
+                    </View> :
+                    <View>
+                        <Animated.View style={[styles.titleWrap, transformStyle]}>
+                            <Text style={styles.title}>Products</Text>
+                        </Animated.View>
+                        <AnimatedFlatList
+                            contentContainerStyle={styles.flatListMarginTop}
+                            onScroll={() => {
```

Comment:
It will not work. Use either
```
onScroll={(event) => {
   Animated.event(
       [{ nativeEvent: { contentOffset: { y: this.flatListValue }} }],
       { useNativeDriver: true }
    )(event)
  }}
```
or

```
onScroll={Animated.event(
       [{ nativeEvent: { contentOffset: { y: this.flatListValue }} }],
       { useNativeDriver: true }
    )
}
```